diff --git a/apps/openmw/mwgui/loadingscreen.cpp b/apps/openmw/mwgui/loadingscreen.cpp
--- a/apps/openmw/mwgui/loadingscreen.cpp
+++ b/apps/openmw/mwgui/loadingscreen.cpp
@@ -280,6 +280,8 @@ namespace MWGui
 
     void LoadingScreen::setupCopyFramebufferToTextureCallback()
     {
+        return;
+
         // Copy the current framebuffer onto a texture and display that texture as the background image
         // Note, we could also set the camera to disable clearing and have the background image transparent,
         // but then we get shaking effects on buffer swaps.
diff --git a/apps/openmw/mwgui/windowmanagerimp.cpp b/apps/openmw/mwgui/windowmanagerimp.cpp
--- a/apps/openmw/mwgui/windowmanagerimp.cpp
+++ b/apps/openmw/mwgui/windowmanagerimp.cpp
@@ -1718,7 +1718,7 @@ namespace MWGui
 
     bool WindowManager::getCursorVisible()
     {
-        return mCursorVisible && mCursorActive;
+        return mCursorVisible;
     }
 
     void WindowManager::trackWindow(Layout* layout, const WindowSettingValues& settings)
