diff --git a/apps/openmw/CMakeLists.txt b/apps/openmw/CMakeLists.txt
--- a/apps/openmw/CMakeLists.txt
+++ b/apps/openmw/CMakeLists.txt
@@ -187,7 +187,14 @@ target_link_libraries(openmw-lib
 )
 
 if (ANDROID)
     target_link_libraries(openmw-lib EGL android log z)
+
+    if (BUILD_OPENMW)
+        # Force static OSG plugin objects into libopenmw.so on Android.
+        find_package(OSGPlugins REQUIRED COMPONENTS ${USED_OSG_PLUGINS})
+        list(APPEND OPENSCENEGRAPH_LIBRARIES ${OSGPlugins_LIBRARIES})
+        target_link_libraries(openmw ${OPENSCENEGRAPH_LIBRARIES} freetype jpeg png)
+    endif()
 endif (ANDROID)
 
 if (USE_SYSTEM_TINYXML)
