diff --git a/apps/openmw/mwstate/statemanagerimp.cpp b/apps/openmw/mwstate/statemanagerimp.cpp
--- a/apps/openmw/mwstate/statemanagerimp.cpp
+++ b/apps/openmw/mwstate/statemanagerimp.cpp
@@ -19,6 +19,7 @@
 #include <components/settings/values.hpp>

 #include <osg/Image>
+#include <osg/Hint>

 #include <osgDB/Registry>

@@ -336,6 +337,8 @@ void MWState::StateManager::saveGame(std::string_view description, const Slot*
         Settings::saves().mCharacter.set(Files::pathToUnicodeString(slot->mPath.parent_path().filename()));
         mLastSavegame = slot->mPath;

+        osg::Hint::GL4ES_SavePSA();
+
         const auto finish = std::chrono::steady_clock::now();

         Log(Debug::Info) << '\'' << description << "' is saved in "
